api/serializers/personal_info.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+from rest_framework import serializers
+
+from api.models import PersonalInfo
+
+
+class PersonalInfoSerializer(serializers.ModelSerializer[PersonalInfo]):
+    class Meta:
+        model = PersonalInfo
+        fields = "__all__"
+
```

Comment:
`...WriteSerializer`?

Reply:
Not for now I think

Reply:
I'm just trying to understand why you created `..WriteSerializer` in one place and not in another

Reply:
So it's different for `Teacher`, but same for `TeacherUnder18`?

Reply:
Yes, because TeacherUnder18 doesn't have fields which should have different format on reading and writing:
`student_age_ranges, teaching_languages_and_levels, availability_slots`